test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -48,7 +48,7 @@
 MIN_ONNX_OPSET_VERSION = 9
 # The max onnx opset version to test for
 MAX_ONNX_OPSET_VERSION = (
-    _constants.ONNX_MAX_OPSET - 1
+    _constants.ONNX_MAX_OPSET - 2
```

Comment:
opset 18 wasn't enabled for these tests.

tests were failing because opset version was bumped to 19, which made these tests to run with opset 18.
I updated this to be `-2` to keep those test to run only with opset<18

Reply:
There are the `skipIfUnsupportedMinOpsetVersion(min_opset_version)` and `skipIfUnsupportedMaxOpsetVersion(max_opset_version)` if the failing cases are just a few

Reply:
I'm familiar with those decorators, but I wish to keep the previous setup instead of making big changes.
enabling opset18 to run on my dev machine got ~250 failures, many of these related to torch not supporting the export of a specific op at all, or not complying to opset18 spec